packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -439,9 +439,9 @@ class SystemRulesTable extends React.Component {
             result = policyRows;
         }
 
-        if (title.length > 0 && result.length > 0) {
+        if (title && title.length > 0 && result.length > 0) {
```

Comment:
If you currently click on a filter without writing some search, it will fail with `title is undefined` when it gets to this point.